templates/modules/b2b-account/quotes/edit-quote/product-picker-table.hypr.live

```diff
@@ -103,7 +103,10 @@
         {% if item.fulfillmentMethod != "Digital" %}
           <select class="mz-l-formfieldgroup-halfsize" data-mz-value="fulfillmentMethod" data-mz-quote-item="{{ item.id }}" >
             <option {% if item.fulfillmentMethod=="Ship" %} selected {% endif %} value="Ship">Direct Ship</option>
```

Comment:
Can you also take the labels "Direct Ship" and "Pickup" from localizations JSON files?

Reply:
moved them to label file